### PR TITLE
v0.28: Add pagination to core concepts/indexes

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Adds the pagination index setting to:
-`learn/core_concepts/indexes.md`
-`learn/configuration/settings.md`

### CODE REVIEW

File: learn/core_concepts/indexes.md [hunk 1]
```diff
@@ -130,3 +131,11 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance).
 
 [Learn more about typo tolerance](/learn/configuration/typo_tolerance.md)
+
+### Pagination
+
+To protect your database from malicious scraping, Meilisearch returns upto 1000 results for a search query. You can update this using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update pagination settings endpoint](/reference/api/pagination.md#update-pagination-settings).
+
+We recommend using the [`limit`](/reference/api/search.md#limit) and [`offset`](/reference/api/search.md#offset) search parameters to implement previous and next buttons for a pagination UI.
```
Comment (maryamsulemani97) [inline learn/core_concepts/indexes.md:139]: I'm not sure if we need this line here, but this section looked too short compared to the rest

File: learn/core_concepts/indexes.md [hunk 2]
```diff
@@ -130,3 +131,11 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance).
 
 [Learn more about typo tolerance](/learn/configuration/typo_tolerance.md)
+
+### Pagination
+
+To protect your database from malicious scraping, Meilisearch returns upto 1000 results for a search query. You can update this using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update pagination settings endpoint](/reference/api/pagination.md#update-pagination-settings).
```
Comment (guimachiavelli) [inline learn/core_concepts/indexes.md:137]:
```suggestion
To protect your database from malicious scraping, Meilisearch only returns up to 1000 results for a search query. You can change this limit using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update pagination settings endpoint](/reference/api/pagination.md#update-pagination-settings).
```

File: learn/core_concepts/indexes.md (hunk 1 above)
Comment (guimachiavelli) [reply learn/core_concepts/indexes.md:139]: I think we can skip it. I feel it doesn't offer enough information.

Perhaps we could add a "You can read more about paginating with Meilisearch on our dedicated guide" linking to the page that's on the other pagination PR?

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -958,3 +958,10 @@ getting_started_typo_tolerance: |-
     --data-binary '{
       "minWordSizeForTypos": { "oneTypo": 4 }
   }'
+settings_guide_pagination_1: |-
+  curl \
+    -X PATCH 'http://localhost:7700/indexes/movies/settings/pagination' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "maxTotalHits": 50
+  }'
```
Comment (guimachiavelli) [inline .code-samples.meilisearch.yaml:967]:
```suggestion
  }'
  
```

File: .vuepress/public/sample-template.yaml [hunk 4]
```diff
@@ -146,3 +146,4 @@ updating_guide_retrieve_documents_old: |-
 updating_guide_update_settings_old: |-
 updating_guide_add_documents_old: |-
 getting_started_typo_tolerance: |-
+settings_guide_pagination_1: |-
```
Comment (guimachiavelli) [inline .vuepress/public/sample-template.yaml:149]:
```suggestion
settings_guide_pagination_1: |-

```

**Review by guimachiavelli — APPROVED** [review]
Approving this, but adding to returns to the end of the code samples/samples template files because GH complains about it otherwise.

File: .code-samples.meilisearch.yaml [hunk 5]
```diff
@@ -958,3 +958,11 @@ getting_started_typo_tolerance: |-
...
+
```
Comment (guimachiavelli) [inline .code-samples.meilisearch.yaml:968]:
```suggestion

```

**Comment by guimachiavelli** [comment]
Had to change base to address missing link, merge conflicts

bors merge

**Pull request merged**